stdlib/2and3/logging/__init__.pyi

```diff
@@ -27,6 +27,8 @@ else:
 
 raiseExceptions: bool
 
+currentframe = ...  # type: Callable[[], FrameType]
```

Comment:
As this is not really an attribute that is meant to be changed, it's better to write this just as an ordinary function. (Also, we prefer PEP 526 variable annotations for new and changed code, but this is irrelevant if changed to use `def`.)

Reply:
I was debating myself, wasn't sure. I will switch to `def`.